pyworkflow/protocol/executor.py

```diff
@@ -297,7 +299,8 @@ def renameGpuIds(self):
 
         logger.debug("Updated gpus ids rebase starting from 0: %s per thread" %self.gpuDict)
 
-    def runJob(self, log, programName, params, numberOfMpi=1, numberOfThreads=1, env=None, cwd=None, executable=None):
+    def runJob(self, log, programName, params, numberOfMpi=1, numberOfThreads=1, env=None, cwd=None,
```

Comment:
When running the protocol using Slurm or similar the protocol is not going to work.

Reply:
Can we make a note about that somehow?